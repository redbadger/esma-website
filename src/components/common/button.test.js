import React from "react";
import renderer from "react-test-renderer";
import Button from "./button";

describe("Button", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Button variant="dark" label="test" onClick={() => {}} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
