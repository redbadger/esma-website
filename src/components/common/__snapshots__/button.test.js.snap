// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  css={
    Object {
      "map": undefined,
      "name": "vxeuw9",
      "next": undefined,
      "styles": "
  height:3rem;width:22rem;--bg-opacity:1;background-color:rgba(45, 49, 66, var(--bg-opacity));--text-opacity:1;color:rgba(255, 255, 255, var(--text-opacity));font-weight:700;
",
      "toString": [Function],
    }
  }
  onClick={[Function]}
>
  test
</button>
`;
